examples/hackweek_demo/extract_data_logs.py

```diff
@@ -26,15 +26,17 @@ def read_json_to_pd(json_file):
     return pd.DataFrame(json_data)
 
 
-def calculate_spot_price_from_state(state, maturity_timestamp, block_timestamp, position_duration):
+def calculate_spot_price_from_state(state, maturity_timestamp, block_timestamp, config_data):
     """Calculate spot price from reserves stored in a state variable."""
     return calculate_spot_price(
         state.shareReserves,
         state.bondReserves,
         state.lpTotalSupply,
         maturity_timestamp,
         block_timestamp,
-        position_duration,
+        position_duration=config_data["positionDuration"],
+        stretch_time=config_data["invTimeStretch"],
```

Comment:
are you sure we want the inverted version here?

Reply:
i think so. the other one was around 22, where the invTimeStretch was closer to the value

Reply:
oh I see what we store as `invTimeStretch` is their `hyperdrive_config["timeStretch"]` that's correct

Reply:
looking at https://github.com/delvtech/elf-simulations/pull/751/files#diff-3a0341d06c9bd4cf543c2823827294caf1cb0188fadd925744a47390051d05b7R180